.licenserc.json

```diff
@@ -94,5 +94,37 @@
     " * See the License for the specific language governing permissions and",
     " * limitations under the License.",
     " */"
+  ],
+  "**/*.ts": [
+    "/* Copyright 2020 Energinet DataHub A/S",
```

Comment:
suggestion: add a `@license` annotation to the beginning of all entries, that is before the copyright note

for example:
```
/**
 * @license
 * Copyright 2021 Energinet DataHub A/S
 * (...)
 */
```